admin/settings/class-balance.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Do we need to name this file with `class` prefix? It doesn't look to me to name a class with that prefix